lua/entities/gmod_wire_hudindicator/init.lua

```diff
@@ -363,11 +363,11 @@ function ENT:Think()
 	return true
 end
 
-// Advanced Duplicator Support
+-- Advanced Duplicator Support
 function ENT:BuildDupeInfo()
 	local info = self.BaseClass.BuildDupeInfo(self) or {}
 
-	if (self.Pod) and (self.Pod:IsValid()) then
+	if self.Pod) and (self.Pod:IsValid() then
```

Comment:
This was very hard to parse. I had to check each occurrence manually before replacing and make sure this didn't happen.